src/Microsoft.Kusto.ServiceLayer/Scripting/ScriptAsScriptingOperation.cs

```diff
@@ -118,13 +113,6 @@ public override void Execute()
                     });
                 }
             }
-            finally
-            {
-                if (disconnectAtDispose && DataSource != null)
-                {
-                    DataSource.Dispose();
-                }
-            }
```

Comment:
Don't dispose of the datasource because it's a singleton from the ConnectionService